tests/js/spec/components/assigneeSelector.spec.jsx

```diff
@@ -136,7 +136,7 @@ describe('AssigneeSelector', function() {
 
   it('should initially have loading state', function() {
     openMenu();
-    expect(assigneeSelector.find('LoadingIndicator')).toHaveLength(1);
+    expect(assigneeSelector.find('LoadingIndicator')).toHaveLength(2);
```

Comment:
Do we really expect to have 2x "LoadingIndicator"?
Is the profiler instance confused with the LoadingIndicator itself? If so we could try to fine tune the selector.